cli/flox/src/commands/environment.rs

```diff
@@ -396,6 +400,12 @@ impl Activate {
             bail!("Unsupported SHELL '{shell}'");
         };
 
+        if !self.run_args.is_empty() {
+            command.arg("-i");
```

Comment:
I think maybe we shouldn't pass `-i`, because I think that's contrary to expectations/what interactive means. But that would require reworking our init scripts a bit

Reply:
Yeah the `-i` was the quickest way to get echt files and all our integration with it working.